docs/md/modules-wgs.md

```diff
@@ -0,0 +1,333 @@
+
+## Usage
+
+ExternalWholeGenomeReprocessing workload supports the following API endpoints:
+
+| Verb | Endpoint                     | Description                                                    |
+|------|------------------------------|----------------------------------------------------------------|
+| GET  | `/api/v1/workload`           | List all workloads                                             |
+| GET  | `/api/v1/workload/{uuid}`    | Query for a workload by its UUID                               |
+| GET  | `/api/v1/workload/{project}` | Query for a workload by its Project name                       |
+| POST | `/api/v1/create`             | Create a new workload                                          |
+| POST | `/api/v1/start`              | Start a workload                                               |
+| POST | `/api/v1/exec`               | Create and start (execute) a workload                          |
+
+### Create Workload: `/api/v1/create`
+Creates a WFL workload. Before processing, confirm that the WFL and Cromwell service accounts have
+at least read access to the input files.
+
+=== "Request"
+
+    ```bash
+    curl --location --request POST 'https://dev-wfl.gotc-dev.broadinstitute.org/api/v1/create' \
+    --header 'Authorization: Bearer '$(gcloud auth print-access-token) \
+    --header 'Content-Type: application/json' \
+    --data-raw '{
+      "cromwell": "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org",
+      "input": "gs://broad-gotc-dev-wfl-ptc-test-inputs/single_sample/plumbing/truth",
+      "output": "gs://broad-gotc-dev-wfl-ptc-test-outputs/wgs-test-output/",
+      "pipeline": "ExternalWholeGenomeReprocessing",
+      "project": "PO-1234",
+      "items": [
+        {
+          "inputs": {
+            "input_cram": "develop/20k/NA12878_PLUMBING.cram",
+            "sample_name": "TestSample1234"
+          }
+        }
+      ]
+    }'
+    ```
+
+=== "Response"
+
+    ```json
+    {
+      "creator": "sehsan@broadinstitute.org",
+      "pipeline": "ExternalWholeGenomeReprocessing",
+      "cromwell": "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org",
+      "release": "ExternalWholeGenomeReprocessing_v1.0",
+      "created": "2020-10-05T15:50:01Z",
+      "output": "gs://broad-gotc-dev-wfl-ptc-test-outputs/wgs-test-output/",
+      "project": "PO-1234",
+      "id": 30,
+      "commit": "d65371ca983b4f0d4fa06868e2946a8e3cab291b",
+      "wdl": "pipelines/reprocessing/external/wgs/ExternalWholeGenomeReprocessing.wdl",
+      "input": "gs://broad-gotc-dev-wfl-ptc-test-inputs/single_sample/plumbing/truth",
+      "uuid": "74d96a04-fea7-4270-a02b-a319dae2dd5e",
+      "items": "ExternalWholeGenomeReprocessing_000000030",
+      "version": "0.3.2"
+    }
+    ```
+
+Note that the ExternalWholeGenomeReprocessing pipeline supports specifying
+cromwell "workflowOptions" via the `options` map. See the
+[reference page](./workflow-options.md) for more information.
```

Comment:
For my own sanity, why does `/docs/md/workflow-options.md` work in the exomes doc but not here?

Reply:
That's a really good point, and a great catch, since I didn't notice the link in exomes doc, that's also broken.. will fix!